### DISCUSSION TITLE
[Errno 13] Permission denied when generate CSV report on docker

### DISCUSSION THREAD

**Discussion by RifkiNurgraha** [ludeknovy/jtl-reporter#81] [Q&A] [answered]
Hi i need help about run locust and integrate jtl report on docker, but when i run locust in docker can't generate the csv. [Errno 13] Permission denied. 
![image](https://user-images.githubusercontent.com/39902589/124058582-d96ef380-da53-11eb-8382-f791c1ffc3f6.png)

This is my dockerfile:

FROM locustio/locust
WORKDIR /code
COPY . .
ENV PYTHONPATH="/code:$PYTHONPATH"
RUN pip install bs4

i know this is out of context but i really need help regarding this 😂. Thank you

**Comment by ludeknovy** [comment, marked as answer]
Hi @RifkiNurgraha, yes, this is not issue of jtl reporter ;).

Have a look at this: https://vsupalov.com/docker-shared-permissions/

**Reply by ludeknovy** [reply]
@RifkiNurgraha did you solve it?

**Reply by RifkiNurgraha** [reply]
yes solved, thank youuu
